components/prospects/ProfileInfo.tsx

```diff
@@ -15,8 +18,17 @@ function ProfileInfo({ prospect }) {
     prospect.phone = phoneUtil.format(phoneNumber, PNF_NATIONAL)
   }
 
+  function deleteProfile() {
+    Router.push('/prospects/list')
```

Comment:
Let's add a `// TODO:` comment here where we specify that the prospect should also be deleted from the DB.